src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rent = 40;
 
+  if (days < 3) {
+    return rent * days;
+  } else
+  if (days >= 3 && days < 7) {
+    return (rent * days) - 20;
+  } else
+  if (days >= 7) {
+    return (rent * days) - 50;
+  }
```

Comment:
use early return pattern
```suggestion
  if (days < 3) {
    return rent * days;
  } 
  
  if (days >= 3 && days < 7) {
    return (rent * days) - 20;
  } 
  
  return (rent * days) - 50;

```